kubernetes/ingress/supernovahunters.org.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: supernovahunters-org-tls
+  labels:
+    use-azuredns-solver: "true"
+spec:
+  issuerRef:
+    name: letsencrypt-prod
+    kind: ClusterIssuer
+  secretName: supernovahunters-org-tls
+  dnsNames:
+    - supernovahunters.org
+    - "*.supernovahunters.org"
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: supernovahunters.org
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
+    nginx.ingress.kubernetes.io/set-real-ip-from: "10.0.0.0/8"
+spec:
+  tls:
+  - hosts:
+    - supernovahunters.org
+    - "*.supernovahunters.org"
+    secretName: supernovahunters-org-tls
+  rules:
+  - host: supernovahunters.org
+    http:
+      paths:
+      - backend:
+          serviceName: http-frontend
+          servicePort: 80
+        path: /(.*)
```

remove the regex linked to the rewrite target as we don't use it
```suggestion
        path: /
```